### PR TITLE
[Enhancement](Load) make consumer number of routine load task configurable

### PR DESCRIPTION
**Pull request by GoGoWen**
## Proposed changes

make consumer number of routine load task configurable

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
I think config max_consumer_num_per_group is same with consumer_num_per_task.

**Comment by GoGoWen** [comment]
> max_consumer_num_per_group

max_consumer_num_per_group for BE scope,  consumer_num_per_task for job scope.

**Comment by HHoflittlefish777** [comment]
> > max_consumer_num_per_group
> 
> max_consumer_num_per_group for BE scope, consumer_num_per_task for job scope.

It means that you want to control it in create routine load job stmt?

**Comment by GoGoWen** [comment]
> > > max_consumer_num_per_group
> > 
> > 
> > max_consumer_num_per_group for BE scope, consumer_num_per_task for job scope.
> 
> It means that you want to control it in create routine load job stmt?

correct!

**Comment by GoGoWen** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java [hunk 1]
```diff
@@ -86,9 +86,11 @@ public final class FeMetaVersion {
     public static final int VERSION_132 = 132;
     // for expr serde
     public static final int VERSION_133 = 133;
+    // add config for consumer number of routine load task
+    public static final int VERSION_134 = 134;
 
     // note: when increment meta version, should assign the latest version to VERSION_CURRENT
-    public static final int VERSION_CURRENT = VERSION_133;
```
Comment (dataroaring) [inline fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java:91]: Why do you change it?

**Review by dataroaring — CHANGES_REQUESTED** [review]
Please add a case and update doc.

**Pull request closed without merging**